cairo/src/font/user_fonts.rs

```diff
@@ -41,12 +45,14 @@ impl UserFontFace {
             extents: *mut ffi::cairo_font_extents_t,
         ) -> ffi::cairo_status_t {
             let init_func = INIT_FUNC.get().unwrap();
+            let mut font_extents = FontExtents::from(*extents);
```

Comment:
```suggestion
            let font_extents = &mut *(extents as *mut FontExtents);
```

Would also work FWIW (because `#[repr(transparent)]`). No need to copy it around